### PR TITLE
Update changelog for 2.6-alpha

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## [2.6.0](https://github.com/mixxxdj/mixxx/milestone/44) (Unreleased)
 
+### SETM support
+
+* Add simple support for STEM files [#12744](https://github.com/mixxxdj/mixxx/pull/12744)
```
Comment (acolombier) [inline CHANGELOG.md:7]: Looks like the wrong issue. Did you mean to point to #13044 ?

**Review by acolombier — CHANGES_REQUESTED** [review]
Just for my knowledge, is that something we should aim to update as part of the PR itself or is that something we are happy to update once in a while with a separated PR?

File: CHANGELOG.md (hunk 1 above)
Comment (daschuer) [reply CHANGELOG.md]: Thank you for noticing. Fixed!

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
 ## [2.6.0](https://github.com/mixxxdj/mixxx/milestone/44) (Unreleased)
 
+### Waveforms
+
+* Simplify waveform combobox in preferences
+  [#13220](https://github.com/mixxxdj/mixxx/issues/13220)
+  [#6428](https://github.com/mixxxdj/mixxx/issues/6428)
+  [#13226](https://github.com/mixxxdj/mixxx/issues/13226)
+
+### SETM support
```
Comment (ronso0) [inline CHANGELOG.md:12]:
```suggestion
### STEM support
```

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Comment by daschuer** [comment]
Fixed and updated.

**Comment by ronso0** [comment]
Thanks! I'd use PascalCase fro scoped timer but we can clean up stuff like that later on.

**Pull request merged**